docs/orchestration/flow_config/run_configs.md

```diff
@@ -58,6 +58,22 @@ register a flow with no labels it will only be picked up by Agents which also
 do not have labels specified.
 :::
 
+:::tip Automatic Labels
+If your Flow is configured for [Local Storage](storage.md#local), which is the
+default, then flow runs generated from it will automatically be given a label
+matching the hostname of the machine where the flow run was created. This behavior can be disabled
+by adding the following to your `config.toml`:
+
+    [flows.defaults]
+        [flows.defaults.storage]
+
+        add_default_labels = false
+
+However, doing this does not enable you to use Local Storage with any agent
+besides the [local agent](/orchestration/agents/local.md), which expects
+the hostname label to be there by default.
```

Comment:
This blurb is strictly true but in order to be useful it needs to provide more context on why this behavior is currently implemented -- the auto-labeling is a feature to prevent non-local agents from submitting flows that are stored in a local filesystem.  

Using `LocalStorage` + a non-local agent is a supported pattern but requires additional setup (e.g., mounting volumes) that are agent and platform dependent.  This should be called out here so that people don't turn off autolabeling without understanding what additional work will be required.